node/npm/glide/index.ts

```diff
@@ -147,6 +147,7 @@ function initialize() {
         ScoreFilter,
         SignedEncoding,
         UnsignedEncoding,
+        ZScanOptions,
```

Comment:
You don't define that type. I think we need to export `ZScanOptions`, but not `BaseScanOptions`.